kinogoteborg/src/app/components/booking/amountOfGuests.jsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+
+import { useEffect, useState } from "react";
+
+export default function AmountOfGuests({ seats, setSeats, selectedTime, selectedDate }) {
+  const [numberOfGuests, setNumberOfGuests] = useState([]);
+
+  useEffect(() => {
+    const fetchAmountOfGuests = () => {
```

Comment:
you dont seem to need the function wrapper since your not using anyting thats async, confirmed by testing.

more for knowledge then require to change.

Reply:
Yes that might be correct. The intention is to use useEffect() and handle all logic in the component-file and keep the same style of the code as the rest of the screening components. But with const instead of useEffect it had been less lines of code in the component-file I think.